_posts/2025-02-01-next-scala-lts.md

```diff
@@ -0,0 +1,117 @@
+---
+category: announcement
+permalink: /news/next-scala-lts.html
+title: "Next Scala 3 LTS version"
+by: Tomasz Godzik
+---
+
+**TLDR;**
+
+API used in lazy val implementation, will be removed in a future JDK thus one of
+the Scala 3 Next minors in 2025 Q4 and the next LTS will drop JDK 8 support.
+
+## Intro
+
+For the compiler’s second LTS version, the Scala 3 compiler team will be
+dropping support for JDK 8 and is considering which later JDK to use.
```

Comment:
Scala 2.12.x and 2.13.x will continue to support Java 8 I think.

Reply:
That's for sure, we can't drop support for a patch version